contents/docs/user-guides/funnels.md

```diff
@@ -138,7 +138,25 @@ You can instantly see that the conversion rate is lower on Linux; this indicates
 
 You can break down by a number of user and event properties (e.g. browser, country, etc.) to help quickly validate the initial hypothesis you may have.
 
-Automating the solution to highlight potential problems and hypotheses for you is something we're actively working on so, take a look at [our roadmap](/handbook/strategy/roadmap) to learn more about what we're building here.
+### Automatically identify possible causes of failure and success
```

Comment:
I think this warrants it's own feature page (adding some link from funnels to the new page).